lstests/core/__init__.py

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env python3
```

Comment:
Remove commented code.  `__init__.py` can be empty and will still invoke package handling behavior.

Reply:
Removed everything. `__init__.py` is now an empty file. Thank you.